### PR TITLE
carineoliveira/c03-docker02

### PR DESCRIPTION
**Pull request by CarineOliveira**
# Description

Exercise c03-docker02

Student Name: Carine Oliveira

> A brief explanation of what has been done.

## Type of change

- [ x ] Exercise Submission (one exercise per PR)
- [ x ] This PR is just for **one** exercise of a class
- [ x ] PR name follows the pattern `<github-username>/<exercise-number>`
- [ x ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker02/carineoliveira/Dockerfile.v2 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM httpd:2.4.41-alpine 
+EXPOSE 80
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker02/carineoliveira/Dockerfile.v2:2]: the /usr/local/apache2/htdocs/ must be empty and the image being used already contain files there.

File: classes/03class/exercises/c03-docker02/carineoliveira/index.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Class 03 - Exercise Docker 02 </h1>
+<h2>V1</h2>
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker02/carineoliveira/index.html:2]: this file does not need to be included in the PR

File: classes/03class/exercises/c03-docker02/carineoliveira/Dockerfile.v2 (hunk 1 above)
Comment (CarineOliveira) [reply classes/03class/exercises/c03-docker02/carineoliveira/Dockerfile.v2]: thanks, added the command on the last commit.

File: classes/03class/exercises/c03-docker02/carineoliveira/index.html (hunk 2 above)
Comment (CarineOliveira) [reply classes/03class/exercises/c03-docker02/carineoliveira/index.html]: removed on previous commit.

**Pull request merged**